README.md

```diff
@@ -1,6 +1,6 @@
 # OctoPrint-docker 
 
-The primary image of `octoprint/octoprint`, is designed to work similarly, and support the
+The primary image of `octoprint/octoprint`. It is designed to work similarly, and support the
```

Comment:
This changes the sentence into a sentence fragment.  Can you fix that?

Reply:
Done.